website/source/downloads_tools.html.erb

```diff
@@ -109,6 +109,9 @@ description: |-
   <li>
     <a href="https://github.com/spring-cloud/spring-cloud-consul">Spring Cloud Consul</a> - Service discovery, configuration and events for Spring Cloud
   </li>
+  <li>
+    <a href="https://github.com/criteo/consul-templaterb/">consul-templaterb</a> - Ruby <a href="https://rubygems.org/gems/consul-templaterb">GEM</a> and executable hi-performance ERB templating to create configuration files, HTML, JSON, XML content from Consul discovery and KV Data.
```

Comment:
Just a pedantic nit here -- the Ruby community usually lowercases `gem` from what I've seen.